.travis.yml

```diff
@@ -1,47 +1,46 @@
 language: node_js
 node_js:
-    - "7.2.1"
-# Selenium-fixes, see: https://github.com/dwyl/learn-nightwatch/issues/8#issuecomment-264734792
+- 7.2.1
 dist: trusty
 sudo: required
 before_install:
- - export CHROME_BIN=/usr/bin/google-chrome
- - export DISPLAY=:99.0
- - sh -e /etc/init.d/xvfb start
- - sudo apt-get update
- - sudo apt-get install -y libappindicator1 fonts-liberation
- - wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
- - sudo dpkg -i google-chrome*.deb
+- export CHROME_BIN=/usr/bin/google-chrome
+- export DISPLAY=:99.0
+- sh -e /etc/init.d/xvfb start
+- sudo apt-get update
+- sudo apt-get install -y libappindicator1 fonts-liberation
+- wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+- sudo dpkg -i google-chrome*.deb
 before_script:
-  - jdk_switcher use oraclejdk8
+- jdk_switcher use oraclejdk8
 after_success:
-  - npm run build
-
+- npm run build
 deploy:
-    - provider: s3
-      access_key_id: ${$AWS_ACCESS_KEY_ID}
-      secret_access_key: ${AWS_SECRET_ACCESS_KEY}
-      bucket: "mpower.sagebridge.org"
-      skip_cleanup: true
-      acl: public_read
-      detect_encoding: false
-      endpoint: mpower.sagebridge.org.s3-website-us-east-1.amazonaws.com
-      region: "us-east-1"
-      local_dir: dist
-      on:
-          branch: develop
-    - provider: s3
-      access_key_id: ${$AWS_ACCESS_KEY_ID}
-      secret_access_key: ${AWS_SECRET_ACCESS_KEY}
-      bucket: "parkinsonmpower.org"
-      skip_cleanup: true
-      acl: public_read
-      detect_encoding: false
-      endpoint: parkinsonmpower.org.s3-website-us-east-1.amazonaws.com
-      region: "us-east-1"
-      local_dir: dist
-      on:
-          branch: release
+  matrix:
```

Comment:
You should remove this.  i think this is added by the travis client when you pass '--add' param to travis encrypt command.  i think that is also the reason for all the formatting changes.  I usually don't pass the '--add' param but follow the instructions to manually update the .travis.yml file with the secret key.